Dashboard/app/js/lib/components/devices/AssignmentsEditView/__partials/DeviceEmpty.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class DeviceEmpty extends React.Component {
+  render() {
+    return (
+      <div className="device-empty">
+        <i className="fa fa-mobile" />
+        <p>{this.props.warningText}</p>
```

Comment:
```suggestion
        <p>{this.props.strings.warningText}</p>
```

Is it part of translated strings?

Reply:
No not a translation. A string will be passed into the component, so it can display whatever info you want.